### PR TITLE
Added command line option for log-level

### PR DESCRIPTION
**Pull request by TwoOfTwelve**
Added a cli option to set the log-level

### CODE REVIEW

**Comment by Kr0nox** [comment]
Do we want the progress bars on level that shows less than info? (WARN, ERROR)
Do we want to keep the progress bars on DEBUG and TRACE? They are barely visible anyway and due to #1719 the additional messages are cut off sometimes

File: cli/src/main/java/de/jplag/cli/logger/JPlagLoggerBase.java [hunk 1]
```diff
@@ -16,14 +16,12 @@ public abstract class JPlagLoggerBase extends AbstractLogger {
 
     private static final Level LOG_LEVEL_FOR_EXTERNAL_LIBRARIES = LOG_LEVEL_ERROR;
 
-    private final Level currentLogLevel;
+    public static Level currentLogLevel = LOG_LEVEL_INFO;
```
Comment (tsaglam) [inline cli/src/main/java/de/jplag/cli/logger/JPlagLoggerBase.java:19]: I would prefer to keep this private and use a static setter instead.

**Review by tsaglam — CHANGES_REQUESTED** [review]
One small thing and a sonar issue.

**Review by Kr0nox — CHANGES_REQUESTED** [review]
Readding my comment from above as requested changes, to not forget it before merging:
> Do we want the progress bars on level that shows less than info? (WARN, ERROR) Users that select these levels probably actually only want warnings and errors

> Do we want to keep the progress bars on DEBUG and TRACE? They are barely visible anyway and due to https://github.com/jplag/JPlag/issues/1719 the additional messages are cut off sometimes

In the meeting we decided we dont want the bar on any level other than info

**Comment by tsaglam** [comment]
One conflict has to be solved before I can merge.

**Review by Kr0nox — APPROVED** [review]

**Review by tsaglam — APPROVED** [review]

**Pull request merged**